apps/marketplace/components/Footer/AUFooter.js

```diff
@@ -34,20 +34,27 @@ const AUFooter = () => (
           <div className="au-footer__end">
             <div className="footer__content footer__legal">
               <p>
-                An initiative of the <a href="https://www.dta.gov.au/">Digital Transformation Agency</a>.{' '}
-                This program forms part of the National Innovation and Science <a href="https://innovation.gov.au/page/agenda">Agenda</a>.{' '}
-                With the exception of the Commonwealth Coat of Arms and where otherwise noted,{' '}
-                this work is licensed under the <a href="https://github.com/govau/uikit/blob/master/LICENSE" rel="external">MIT license</a>
+                An initiative of the <a href="https://www.dta.gov.au/">Digital Transformation Agency</a>. This program
```

Comment:
assume we dont want these changes to be merged?

Reply:
Yes, that is just prettier linting